.github/workflows/dapr-base-containers.yaml

```diff
@@ -8,14 +8,18 @@ name: dapr-base-containers
 on:
   repository_dispatch:
     types: [windows-base]
+  pull_request:
```

Comment:
Do we really need to build these images on every PR?

Reply:
I can revert this, but it was helpful to verify the build works.